### DISCUSSION TITLE
Breadcrumb component in v9

### DISCUSSION THREAD

**Discussion by djayMehrotra** [microsoft/fluentui#24315] [General] [answered]
Will there be a breadcrumb component in future of **fluent ui v9**?

**Comment by layershifter** [comment, marked as answer]
@djayMehrotra it will be implemented, but there is no ETA as it have not been placed to [current roadmap](https://github.com/microsoft/fluentui/wiki/Fluent-UI-React-v9-Component-Roadmap) yet.
